### DISCUSSION TITLE
Add Lexical-based editor to payload's core

### DISCUSSION THREAD

**Discussion by AlessioGr** [payloadcms/payload#1772] [Feature Requests & Ideas]
Just leaving this here for documentation purposes, so you can also prioritize it in the roadmap if you do plan to invest time into it eventually.

Ref: https://github.com/AlessioGr/payload-plugin-lexical and https://github.com/payloadcms/payload/pull/1443

**Comment by jakehopking** [comment]
I can't say I'm a big fan of slateJS, so I'll add my voice to this feature request 👍
